src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -27,39 +32,70 @@
  */
 public class ConcurDetailLineGroupForCollector {
 
+    protected static final String ORPHANED_CASH_ADVANCE_MESSAGE_FRAGMENT = "Cash Advance did not have a corresponding entry from the Request Extract.";

```

Comment:
Yes, it would be good to move these to the application resources eventually. If you think this is critical enough to have it done now, then I can do it for this PR; otherwise, we can defer it to a future clean-up effort.